src/commands/cmd_json.cc

```diff
@@ -616,6 +642,7 @@ REDIS_REGISTER_COMMANDS(MakeCmdAttr<CommandJsonSet>("json.set", 4, "write", 1, 1
                         MakeCmdAttr<CommandJsonObjLen>("json.objlen", -2, "read-only", 1, 1, 1),
                         MakeCmdAttr<CommandJsonStrAppend>("json.strappend", -3, "write", 1, 1, 1),
                         MakeCmdAttr<CommandJsonStrLen>("json.strlen", -2, "read-only", 1, 1, 1),
-                        MakeCmdAttr<CommandJsonMGet>("json.mget", -3, "read-only", 1, 1, 1), );
+                        MakeCmdAttr<CommandJsonMGet>("json.mget", -3, "read-only", 1, 1, 1),
+                        MakeCmdAttr<CommandJsonMSet>("json.mset", -4, "write", 1, 1, 1), );
```

Comment:
```suggestion
                        MakeCmdAttr<CommandJsonMGet>("json.mget", -3, "read-only", 1, -2, 1),
                        MakeCmdAttr<CommandJsonMSet>("json.mset", -4, "write", 1, -1, 3), );
```

Reply:
Thank you very much, but I'm still a little confused about this, is this how it's supposed to be?

```C++
MakeCmdAttr<CommandJsonMGet>("json.mget", -3, "read-only", 1, -2, 1)
MakeCmdAttr<CommandJsonMSet>("json.mset", -4, "write", 1, -3, 3)
```

This is the syntax of json.mset.

```shell
JSON.MSET key path value [key path value ...]
```

Reply:
Yes, that's correct too. Since the key step is 3, there is little difference between -1 and -3 for the stop index.